docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -254,6 +254,8 @@ helm install --namespace kube-system --name cinder-csi ./charts/cinder-csi-plugi
 | StorageClass `parameters`  | `availability`          | `nova`          | String. Volume Availability Zone |
 | StorageClass `parameters`  | `type`                  | Empty String    | String. Name/ID of Volume type. Corresponding volume type should exist in cinder     |
 | VolumeSnapshotClass `parameters` | `force-create`    | `false`         | Enable to support creating snapshot for a volume in in-use status |
+| VolumeSnapshotClass `parameters` | `type`            | Empty String    | `snapshot` creates a VolumeSnapshot object linked to a Cinder volume snapshot. `backup` creates a VolumeSnapshot object linked to a cinder volume backup. Defaults to `snapshot` if not defined |
+| VolumeSnapshotClass `parameters` | `backup-max-duration-per-gb`  | `5`    | Defines the amount of time to wait for a backup to complete per volume size in GB |
```

Comment:
duration but what is the unit? seconds? minutes?

Reply:
if that is seconds, it should be quite fast. 200MB/s, well perhaps worth to use something like 50MB/s by default, which means 20 default value?

Reply:
Done :)